test/shared/src/main/scala/zio/test/TestDuration.scala

```diff
@@ -0,0 +1,50 @@
+package zio.test
+
+import zio._
+
+import java.time.Instant
+
+sealed trait TestDuration { self =>
+  import TestDuration._
+
+  final def <>(that: TestDuration): TestDuration =
+    (self, that) match {
+      case (Zero, Zero)  => Zero
```

Comment:
This case seems redundant.

Reply:
Yes. I was thinking of enumerating all four possible cases but you're right.